### ISSUE TITLE
Isn't this like totally illegal ???

### ISSUE THREAD

**Issue by kenan1099** [TotallyTobyFox/undertale#8]
Did you even seek permission ?

**Comment by aawood** [comment]
Yeah, TTB, unless the person who sent you this was Toby Fox, that they said you could share it doesn't mean anything: You cannot share this. (In fact, the person who sent it you is probably just using you, as now it's you that'll get in trouble if this legal action winds up being taken). Do yourself a favour, drop this repo.

**Comment by Jan200101** [comment]
to more directly answer the issue
yes
this is VERY **VERY** illegal and could have the repo owner get into serious legal trouble, as mentioned by aawood

**Comment by TotallyTobyFox** [comment]
`Do wat yee want Caz a pirate is free`
